codersdk/experiments.go

```diff
@@ -0,0 +1,48 @@
+package codersdk
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+)
+
+var (
+	// ExperimentVSCodeLocal enables a workspace button to launch VSCode
+	// and connect using the local VSCode extension.
+	ExperimentVSCodeLocal = "vscode_local"
```

Comment:
Consider adding one more type, `type Experiment string` for type safety, in which case:
```suggestion
	ExperimentVSCodeLocal Experiment = "vscode_local"
```